### ISSUE TITLE
/settings Slash Commands don't show an error when the user doesn't have sufficient permissions

### ISSUE THREAD

**Issue by Zukunftsmusik** [PieInTheSky-Inc/YaDc#292]
The bot just doesn't respond.

**Comment by Zukunftsmusik** [comment]
Full exception message:
```
2023-01-23T12:47:14.204300708Z 20230123 12:47:14 [ERROR ] root: You are missing Manage Server permission(s) to run this command.
2023-01-23T12:47:14.204350911Z NoneType: None
2023-01-23T12:47:14.204358091Z Ignoring exception in on_application_command_error
2023-01-23T12:47:14.204668228Z Traceback (most recent call last):
2023-01-23T12:47:14.205459455Z File "/usr/local/lib/python3.11/site-packages/discord/client.py", line 377, in _run_event
2023-01-23T12:47:14.205498563Z await coro(*args, **kwargs)
2023-01-23T12:47:14.205509930Z File "/usr/bot/src/bot.py", line 200, in on_application_command_error
2023-01-23T12:47:14.205520411Z help_args = ctx.message.clean_content.replace(command_args, '').strip()[1:]
2023-01-23T12:47:14.205529524Z ^^^^^^^^^^^^^^^^^^^^^^^^^
2023-01-23T12:47:14.205538396Z AttributeError: 'NoneType' object has no attribute 'clean_content'
```

**Comment by Zukunftsmusik** [comment]
Fixed with v1.4.0.5
